soc/nxp/common/nxp_nbu.c

```diff
@@ -0,0 +1,59 @@
+/*
```

Comment:
Please move this driver to `drivers/interrupt_controller` folder. That seems a better place for this.

Reply:
it's not a driver, so no

Reply:
The binding file is listed as interrupt controller. Do we even need a binding file in dts?

Reply:
it could go in dts/arm I guess